HW_1/task_2.py

```diff
@@ -0,0 +1,11 @@
+'''Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+ Сколько журавликов сделал каждый ребенок, 
+если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+*Пример:*
+6 -> 1  4  1
+24 -> 4  16  4
+    60 -> 10  40  10'''
+
+sum = int(input('Введите количество журавликов: '))
+print('Петя', sum // 3 // 2, 'Катя', sum // 3 * 2, 'Сережа', sum // 3 // 2)
```

Comment:
Вот повторяется одно и то же действие: `sum // 3`. Чтобы такого не было создавайте переменные.
```suggestion
print('Петя', var := sum // 3 // 2, 'Катя', var  * 4, 'Сережа', var)
```
Ну и не помешало бы проверять входные данные на кратность 6.